optimum/exporters/onnx/__main__.py

```diff
@@ -221,13 +221,24 @@ def main_export(
             " and passing it is not required anymore."
         )
```

Comment:
also could you update the documentation in https://github.com/huggingface/optimum/blob/b865809b76fff886df5f23e23b1da8f6f24a78b0/docs/source/exporters/onnx/usage_guides/export_a_model.mdx?plain=1#L90 and  https://github.com/huggingface/optimum/blob/b865809b76fff886df5f23e23b1da8f6f24a78b0/docs/source/exporters/tflite/usage_guides/export_a_model.mdx#L62